code/modules/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -1,4 +1,12 @@
 /mob/living/carbon/human/proc/get_unarmed_attack(var/mob/living/carbon/human/target, var/hit_zone)
+	if(src.gloves)

```

Comment:
Might want to do a typecheck here for proper gloves items(?); there's at least one other item that can fit in that slot and it threw/throws runtimes on some other interactions.

Reply:
Done, hopefully should catch it.